jin/(6월 1주차 과제)/10984.py

```diff
@@ -0,0 +1,12 @@
+t = int(input())
+for _ in range(t) :
+  t2 = int(input())
+  c_value = 0
+  g_value = 0
+  for i in range(t2) :
+    c, g = map(str, input().split())
```

Comment:
```suggestion
    c, g = map(float, input().split())
```
값을 처음부터 실수형으로 받고 c_value만 정수형으로 출력하면 타입캐스팅 횟수를 줄일 수 있습니다 ㅎㅎ